fe/fe-core/src/main/java/com/starrocks/catalog/ListPartitionInfo.java

```diff
@@ -152,17 +202,19 @@ private String singleListPartitionSql(OlapTable table, short tableReplicationNum
         return StringUtils.removeEnd(sb.toString(), ",\n");
     }
 
+    private String valuesToString(List<LiteralExpr> values) {
+        return "(" + values.stream().map(value -> "\'" + value.getStringValue() + "\'")
+                .collect(Collectors.joining(", ")) + ")";
+    }
+
     private String multiListPartitionSql(OlapTable table, short tableReplicationNum) {
         StringBuilder sb = new StringBuilder();
-        idToMultiValues.forEach((partitionId, multiValues) -> {
+        this.idToMultiLiteralExprValues.forEach((partitionId, multiValues) -> {
             Short partitionReplicaNum = table.getPartitionInfo().idToReplicationNum.get(partitionId);
             Optional.ofNullable(table.getPartition(partitionId)).ifPresent(partition -> {
                 String partitionName = partition.getName();
                 sb.append("  PARTITION ").append(partitionName).append(" VALUES IN (");
-                String items = multiValues.stream()
-                        .map(values -> "(" + values.stream().map(value -> "\'" + value + "\'")
-                                .collect(Collectors.joining(",")) + ")")
-                        .collect(Collectors.joining(","));
+                String items = this.multiValuesToString(multiValues);
                 sb.append(items);
                 sb.append(")");
```

Comment:
Remove `()` here and put it into `multiValuesToString`.

Consistent with `valuesToString`.